include/hobbes/util/stream.H

```diff
@@ -72,13 +72,19 @@ template < typename Char = char, typename Traits = std::char_traits<Char> >
 
 }}
 
+#define ABS_INT128_POS ((static_cast<unsigned __int128>(1) << 127) - 1)
+#define ABS_INT128_NEG ( static_cast<unsigned __int128>(1) << 127)
+#define INT128_MIN     (static_cast<__int128>(static_cast<unsigned __int128>(1) << 127))
```

Comment:
let's prefix this with something hobbes specific to avoid collision

Reply:
done